neoscore/western/tremolo.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from typing import Optional, Union
+
+from neoscore.core.brush import BrushDef
+from neoscore.core.has_music_font import HasMusicFont
+from neoscore.core.music_font import MusicFont
+from neoscore.core.music_text import MusicText
+from neoscore.core.pen import PenDef
+from neoscore.core.point import PointDef
+from neoscore.core.positioned_object import PositionedObject
+from neoscore.western.chordrest import Chordrest
+
+
+class Tremolo(MusicText):
+    """A tremolo marking over a single Chordrest.
+
+    The tremolo indicator accepts either an integer for declaring the number
+    of strokes of a combining (common) tremolo, or a
+    SMuFL glyphname e.g. "pendereckiTremolo".
+
+    If a bridging tremolo is required between two Chordrests,
+    then move the x position with the pos variable.
+    """
+
+    def __init__(
+        self,
+        pos: PointDef,
+        parent: PositionedObject,
+        indication: Union[int, str],
+        font: Optional[MusicFont] = None,
+        brush: Optional[BrushDef] = None,
+        pen: Optional[PenDef] = None,
+    ):
+        """
+        Args:
+            pos: The starting position
+            parent: The parent for the starting position. If no font is given, this or one of its ancestors must
+                implement :obj:`.HasMusicFont`.
+            indication: number of strokes indicated in the tremolo. Must be 1, 2, 3, 4 or 5,
+                or SMuFL glyphname.
```

Comment:
this doc isn't quite right. "number of strokes indicated..." is not the only thing it can be. A better summary sentence might be something like "The type of tremolo to draw, either a stroke count for conventional tremolos (1-5) or an arbitrary SMuFL glyphname."

Reply:
and be sure to update the doc in the classmethod below too